package.json

```diff
@@ -191,12 +191,12 @@
     "@types/chai-as-promised": "7.1.6",
     "@types/debug": "4.1.9",
     "@types/mocha": "10.0.2",
-    "@types/node": "20.4.6",
+    "@types/node": "20.8.6",
```

Comment:
Note on `@types/node` as well that you may report feature that exist on 20 but not on 16, and break on run time.
Let's address this in another PR though as it was previously like so.

Reply:
Ack